modules/oe_showcase_list_pages/tests/src/ExistingSite/ListPagesTest.php

```diff
@@ -148,66 +139,57 @@ public function testCreateListPages() {
     $search_button->click();
-    $this->assertSearchResultsTitle('News list page', 1);
-    $this->assertSearchResults([
+    $this->assertResultsTitle('Results', 1);
+    $this->assertResults([
       'News number 8',
     ]);
 
     // Filter the News results by content.
     $title_input->setValue('This is a News content number 10');
     $search_button->click();
-    $this->assertSearchResultsTitle('News list page', 1);
-    $this->assertSearchResults([
+    $this->assertResultsTitle('Results', 1);
+    $this->assertResults([
       'News number 10',
     ]);
 
     // Filter the News results by introduction.
     $title_input->setValue('This is a News introduction number 11');
     $search_button->click();
-    $this->assertSearchResultsTitle('News list page', 1);
-    $this->assertSearchResults([
+    $this->assertResultsTitle('Results', 1);
+    $this->assertResults([
       'News number 11',
     ]);
 
     // Filter by a mix of introduction and content in shuffled order.
     $title_input->setValue('6 content number introduction');
     $search_button->click();
-    $this->assertSearchResultsTitle('News list page', 1);
-    $this->assertSearchResults([
+    $this->assertResultsTitle('Results', 1);
+    $this->assertResults([
       'News number 6',
     ]);
     // @todo Remove "7" once we have a default order.
     $title_input->setValue('NUMBER 7 this');
     $search_button->click();
-    $this->assertSearchResultsTitle('News list page', 1);
-    $this->assertSearchResults([
+    $this->assertResultsTitle('Results', 1);
+    $this->assertResults([
       'News number 7',
     ]);
 
     // Assert only News nodes are part of the result.
     $title_input->setValue('Event example');
     $search_button->click();
-    $this->assertSearchResultsTitle('News list page', 0);
+    $this->assertResultsTitle('Results', 0);
 
     // Create an Event listing page.
-    $this->drupalGet('node/add/oe_list_page');
-    $page->fillField('Title', 'Event list page');
-    $page->fillField('Source entity type', 'node');
-    $page->fillField('Source bundle', 'oe_sc_event');
-    $page->pressButton('Save');
-
-    $node = $this->getNodeByTitle('Event list page');
-    $this->drupalGet('node/' . $node->id() . '/edit');
-    $page->checkField('Override default exposed filters');
-    $page->checkField('emr_plugins_oe_list_page[wrapper][exposed_filters][oelp_oe_sc_event__location]');
-    $page->checkField('emr_plugins_oe_list_page[wrapper][exposed_filters][oelp_oe_sc_event__title]');
-    $page->checkField('emr_plugins_oe_list_page[wrapper][exposed_filters][oelp_oe_sc_event__oe_sc_event_dates]');
-    $page->pressButton('Save');
-
-    $this->drupalGet('node/' . $node->id());
+    $list_page = $this->createListPage('oe_sc_event', [
+      'oelp_oe_sc_event__location' => 'oelp_oe_sc_event__location',
+      'oelp_oe_sc_event__title' => 'oelp_oe_sc_event__title',
+      'oelp_oe_sc_event__oe_sc_event_dates' => 'oelp_oe_sc_event__oe_sc_event_dates',
```

Comment:
Let's simplify the parameter, then here we can do this:
```suggestion
      'oelp_oe_sc_event__location',
      'oelp_oe_sc_event__title',
      'oelp_oe_sc_event__oe_sc_event_dates',
```
See my other comment below.